dcipher-agents/src/signature_sender/async_signer.rs

```diff
@@ -48,11 +48,10 @@ where
             alg: self.algorithm,
         };
         let sig = self.signer.async_sign(dsigner_req).await?;
-        let sig_cg = CG::Affine::deser(&sig).map_err(|e| Self::Error::Other(e.into()))?;
 
         Ok(SignedSignatureRequest {
             id: req.id,
-            signature: EvmSerialize::ser_bytes(&sig_cg),
```

Comment:
We can probably yeet the `EvmSerialize` trait if it's not used anywhere other than blocklock-agent